tests/www/views/test_views_trigger_dag.py

```diff
@@ -52,7 +52,6 @@ def test_trigger_dag_button(admin_client):
     assert run.run_type == DagRunType.MANUAL
 
 
-@pytest.mark.quarantined
```

Comment:
Should we remove this in main as well?

Reply:
Absolutely

Reply:
OK will send a PR.

Reply:
I can create a PR to main with de-quarantining and we can cherry-pick both PRs instead.

Reply:
This test was flaky for some time, we should probably take a look on how it was doing recently

Reply:
Yep. Looking at the status of it . I have disabled the automated status gathering of Flaky tests some time ago as it was not as useful due to timeouts  - but we should likely come back to this.